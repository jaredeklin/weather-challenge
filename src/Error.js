import React from 'react';

export const Error = () => {
  return <h1>Oh no! Something went wrong. Please try again.</h1>;
};
