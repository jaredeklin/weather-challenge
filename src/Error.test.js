import React from 'react';
import { Error } from './Error';
import { shallow } from 'enzyme';

describe('Error', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Error />);
    expect(wrapper).toMatchSnapshot();
  });
});
