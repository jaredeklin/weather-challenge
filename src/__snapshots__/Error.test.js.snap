// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Error />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Oh no! Something went wrong. Please try again.",
    },
    "ref": null,
    "rendered": "Oh no! Something went wrong. Please try again.",
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Oh no! Something went wrong. Please try again.",
      },
      "ref": null,
      "rendered": "Oh no! Something went wrong. Please try again.",
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
